packer/install-dependencies.sh

```diff
@@ -363,7 +363,10 @@ K8S_VERSIONS="
 1.14.1
 1.13.6
 1.13.5
+1.13.5-azs
+1.12.9
```

Comment:
this shouldn't add back 1.12.9... might be a rebase error?

Reply:
Yup. Fixing it.